### PR TITLE
added time field input in host ride and join ride pages.

### PR DESCRIPTION
**Pull request by eshita241**
added time field input in host ride and join ride pages, also profile page certain parts are done. #issue 26

### CODE REVIEW

File: components/Time.tsx
Comment (RohanKhatua) [inline components/Time.tsx:1]: change to time_display or something, do not name files after reserved classes

**Review by RohanKhatua — CHANGES_REQUESTED** [review]
Please see review comments

File: components/Time.tsx
Comment (eshita241) [reply components/Time.tsx:1]: okay, changed the name to time_display

**Review by anandrajaram21 — APPROVED** [review]
lgtm

**Pull request merged**